src/components/modal/addtostudio/modal.scss

```diff
@@ -110,7 +110,7 @@
     overflow: hidden;
     text-overflow: ellipsis;
     white-space: nowrap;
-    font-family: "Helvetica Neue";
+    font-family: "Helvetica Neue", sans-serif;
```

Comment:
Perhaps this more common(ly used throughout 3.0) stack would work? https://github.com/LLK/scratch-www/blob/1f8342b987a6f9c9e62001490acf6db65afc5e52/src/components/modal/comments/modal.scss#L31